app/stores/ada/TrezorSendStore.js

```diff
@@ -0,0 +1,188 @@
+// @flow
+import { action, observable } from 'mobx';
+import { defineMessages } from 'react-intl';
+import TrezorConnect from 'trezor-connect';
```

Comment:
try to pre-initialize using `TrezorConnect.init()`

Benefits:
1. User will have to wail bit lesser for `TrezorConnect API` to respond
2. User can close `TrezorConnect` tab before it's fully initialize, as a result:
   A. sometimes `TrezorConnect API` hangs because of this, so it will be solved
   B. UX will be improved

Reply:
DONE